MatrixSDK/ContentScan/PKEncryptionHelper.swift

```diff
@@ -0,0 +1,50 @@
+// 
+// Copyright 2024 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import MatrixSDKCrypto
+
+@objc public class PKEncryptionHelper: NSObject {
+    @objc public static func getPKMessage(_ message: String, using publicKey: String) -> PKMessageWrapper? {
+        do {
+            let pkEncryption = try PkEncryption.fromBase64(key: publicKey)
+            return PKMessageWrapper(pkMessage: pkEncryption.encrypt(plaintext: message))
+        } catch {
+            MXLog.error("[PKEncryptionHelper] failed to create pkEncryption", context: error)
```

Comment:
Is this error safe to log/submit to Sentry?

Reply:
@poljar not sure, let's ask Damir

Reply:
Yes, it just tells you that the public key is invalid and why.

Reply:
The encrypt call could be outside of the try, since it can't fail once you have a correctly decoded public key.